bio_hansel/qc/checks.py

```diff
@@ -10,6 +10,24 @@
 from ..subtype import Subtype
 
 
+def is_overall_coverage_low(st: Subtype, df: DataFrame, p: SubtypingParams) -> Tuple[Optional[str], Optional[str]]:
+    if not st.are_subtypes_consistent:
```

Comment:
Might be cleaner as:
```python 
if <cond 1> or <cond 2> or <cond 3>:
    ...
```
or each condition on a separate line:
```python 
if <cond 1> \
    or <cond 2> \
    or <cond 3>:
    ...
```

Reply:
I like this! Changing so it's prettier.